autometrics/src/lib.rs

```diff
@@ -186,9 +186,16 @@ pub use autometrics_macros::ResultLabels;
 #[cfg(feature = "prometheus-exporter")]
 pub use self::prometheus_exporter::*;
 
+#[cfg(feature = "prometheus-client")]
+pub use self::tracker::prometheus_client::PROMETHEUS_CLIENT_REGISTRY;
```

Comment:
I'm a little hesitant about adding this to the global crate namespace. An alternative would be to have some module like `integrations::prometheus_client` for library-specific things. What do folks think?